### PR TITLE
`distribute`

### PR DESCRIPTION
**Pull request by edsko**

### CODE REVIEW

File: large-anon/src/Data/Record/Anon/Internal/Advanced.hs [hunk 1]
```diff
@@ -341,6 +343,24 @@ sequenceA' = sequenceA . co
...
+    . fmap toCanonical
+  where
+    indices :: [Int]
+    indices = [0 .. pred (length $ proxy fieldNames (Proxy @r))]
+
+    fieldVec :: m (Canonical f) -> Int -> (m :.: f) Any
+    fieldVec cs idx = Comp $ flip Canon.getAtIndex idx <$> cs
+
+distribute' :: (Functor m, KnownFields r) => m (Record I r) -> Record m r
+distribute' = map (fmap unI . unComp) . distribute
```
Comment (johannesgerer) [inline large-anon/src/Data/Record/Anon/Internal/Advanced.hs:362]: I am never sure about the performance implications, but what would you think about coercing similar to e.g. `sequenceA'` ?

Comment (edsko) [reply large-anon/src/Data/Record/Anon/Internal/Advanced.hs]: Yes, fair enough, have changed it.

**Comment by edsko** [comment]
@johannesgerer Ok, released https://hackage.haskell.org/package/large-anon-0.2.1 . Thanks for your contribution :)

**Comment by johannesgerer** [comment]
Thank you!

**Pull request merged**